File: cloud/log-server/src/main/java/pers/acp/admin/log/LogServerApplication.java
package pers.acp.admin.log;

import org.springframework.boot.SpringApplication;
import pers.acp.spring.cloud.annotation.AcpCloudAtomApplication;

/**
 * @author zhangbin by 09/04/2018 16:11
 * @since JDK 11
 */
@AcpCloudAtomApplication
public class LogServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(LogServerApplication.class, args);
    }

}

File: cloud/log-server/src/main/kotlin/pers/acp/admin/log/LogServerApplication.kt
package pers.acp.admin.log

import io.github.zhangbinhub.acp.cloud.annotation.AcpCloudAtomApplication
import org.springframework.boot.runApplication

/**
 * @author zhangbin by 09/04/2018 16:11
 * @since JDK 11
 */
@AcpCloudAtomApplication
class LogServerApplication

fun main(args: Array<String>) {
    runApplication<LogServerApplication>(*args)
}
